site/_config.yml

```diff
@@ -32,3 +32,7 @@ gems:
   - jekyll-seo-tag
   - jekyll-avatar
   - jekyll-mentions
+
+exclude:
+  - README.md
+  - .gitignore
```

Comment:
@parkr if these files were generated with `jekyll new` by default, we could add them to template of `_config.yml`. Repos with jekyll site have large possibilities contain such files as `README.md`, `LICENSE`, `Rakefile`, `.gitignore`, etc.